### ISSUE TITLE
SE-GSM bonding changes and applying forces

### ISSUE THREAD

**Issue by mbuche** [ZimmermanGroup/molecularGSM#6]
How does a user indicate the bonding changes necessary for SE-GSM? 

Also, in theory, how would SE-GSM respond to a molecule that has no other stable geometry? (the reactant geometry is the only finite energy minima, and there may or may not be transition states).

**Comment by mina-jafari** [comment]
You can use the ISOMERS file. It's a text file with the following format:
NEW
ADD 1  2
BREAK 3  4
ANGLE 1  2  3  30.
TORSION 1  2  3  4  120.
The numbers are indices of the atoms that undergo a bond/angle/torsion change.

Please see this link for more information:
https://sites.lsa.umich.edu/zimmerman-lab/tutorial/surface-growing-string-method/growing-string-calculation/

**Comment by mbuche** [comment]
Thank you for your help.

Another thing I could not find, in some of your group's publications you perform the GSM with a force applied to the molecule. How do you include this force in the GSM? Do you use QChem's EFEI? (I would think to include it in the "qend" file, so then it would attach to either geometry in DE-GSM or just one in SE-GSM)

**Comment by kellyalg** [comment]
The force functionality is currently not checked in to QChem. I suggest you contact our PI about a possible collaboration.

**Comment by mbuche** [comment]
Got it, thank you so much. I will try some things on my own first, then contact your PI to maybe collaborate if I cannot find a way.

**Comment by mbuche** [comment]
I think that one can apply forces with your GSM code as it stands, at least in the DE case. I ran the Diels-Alder reaction again with DE-GSM, both with no force and with a 2nN force applied between the outer carbons on the diene (I retrieved the reactant and product configurations under this force in preliminary optimization jobs). It seems to have worked without any issue. 

I accomplished this using QChem's simple EFEI, discussed at the end of the page [here](http://www.q-chem.com/qchem-website/manual/qchem44_manual/sec-constrained_opt.html).
